assets/js/googlesitekit/datastore/user/date-range.js

```diff
@@ -140,6 +178,19 @@ export const selectors = {
 
 		return dates;
 	},
+
+	/**
+	 * Returns the current reference date, typically today.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state The current data store's state.
+	 * @return {string} The current reference date as YYYY-MM-DD.
+	 */
+	getReferenceDate( state ) {
+		const { referenceDate } = state;
+		return referenceDate;
```

Comment:
Not a big deal but the destructuring here is unnecessary.
```suggestion
		return state.referenceDate;
```